plugins/jira-dashboard-common/src/annotations.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The key of the Jira project to track for this entity
+ *
+ *  @public
+ */
+export const PROJECT_KEY_ANNOTATION = 'jira-dashboard/project-key';
+
+/**
+ * Jira components to track for this entity
+ *  @public
+ */
+export const COMPONENT_ANNOTATION = 'jira-dashboard/component';
```

Comment:
this one's singular but the one below is plural, both support lists - should we just be using singular and then have it as a bonus feature that you can specify multiple anyway?

Reply:
Since the plugin already supports multiple Jira components and filter-ids I think I will just change the name to "components".  Fix this now :)